src/Compilers/Core/Portable/Symbols/IMethodSymbol.cs

```diff
@@ -263,6 +264,14 @@ public interface IMethodSymbol : ISymbol
         /// </summary>
         IMethodSymbol? PartialImplementationPart { get; }
 
+        /// <summary>
+        /// Returns the implementation attributes for the given method symbol.
+        /// </summary>
+        /// <remarks>
+        /// These attributes are not always emitted as attributes.
+        /// </remarks>
+        MethodImplAttributes ImplementationAttributes { get; }
```

Comment:
Is there any concern here about using a type from the framework that Roslyn _targets_? i.e. if coreclr adds a new value and the compiler is compiling a project we might see values that aren't in the enum version for what we're targeting. This is _fine_ in that enums don't have any validation but not sure if it's worth a comment this is a potential scenario that could happen...?

If we have a precedent for this sort of API I defer to whatever we did of course.

Reply:
> Is there any concern here about using a type from the framework that Roslyn targets?

We do this in a few other places as well. Personally, I don't see a huge issue with it: it doesn't feel all that different from compiling against an older version of the Roslyn API and seeing unexpected values in the `SyntaxKind` enum.

Reply:
Sounds good to me then.